knowledge-bases/advance-concepts/README.md

```diff
@@ -0,0 +1,23 @@
+# Chunking Evaluation Strategy for optimum RAG applcation using Amazon Bedrock Knowledge Base
```

Comment:
Change the heading of README file to Advanced Concepts and add your notebook as first one to be listed in the README file.